src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.cs

```diff
@@ -1451,8 +1451,9 @@ public SortKey GetSortKey(string source)
             GlobalizationMode.UseNls ?
                 NlsCreateSortKey(source, options) :
 #if TARGET_BROWSER
+            // JS cannot create locale-sensitive sort key, use invaraint functions instead.
             GlobalizationMode.Hybrid ?
-                throw new PlatformNotSupportedException(GetPNSEText("SortKey")) :
+                InvariantCreateSortKey(source, options) :
```

Comment:
Should these changes be pushed down into IcuCreateSortXx rather than ifdef'ing at the call sites?

Reply:
Do you mean moving the ifdefs to `IcuCreateSortKey` function? Ideally, all HybridGlobalization functions concerning Browser should be in `CompareInfo.WebAssembly.cs`, not `CompareInfo.Icu.cs` but since some iOS's switches are already in *.ICU.cs I think that it's fine to make it consitent.

Reply:
Yes... if every call to IcuCreateSortXx would need this special-casing, then I'd expect it to be part of the function itself rather than at the call site. If it's specific to just this one usage, then the call site is more appropriate.